.github/workflows/build-and-test.yml

```diff
@@ -2,7 +2,7 @@ name: Build & Test
 
 on:
   push:
-    branches-ignore: [master]
+    branches: [master]
```

Comment:
Does this mean the tests will only run on `master` going forward? I'd have thought we want to keep testing PRs

Reply:
The next line (not in diff) has them run on all pull requests.